migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go

```diff
@@ -92,23 +92,23 @@ func (s *migrationTestSuite) TestMigration() {
 	}
 
 	//verify getMigratedReportConfigIfExists function
```

Comment:
Pretty sure test files are skipped but let's be consistent with comment style.

Reply:
Yeah updated them just to be safe

Reply:
Looks like tests are evaluated
```
migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go:118:31: rows.Err must be checked (rowserrcheck)
	configs, err := s.gormDB.Rows()
	                             ^
migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go:120:41: rows.Err must be checked (rowserrcheck)
	snapshots, err := s.snapshotgormdB.Rows()
	                                       ^
```